src/app/user-settings/components/social-networks/social-networks.component.html

```diff
@@ -782,7 +782,7 @@
         >
           <div class="col-1 col-md-1 col-sm-1 bloc-name">
             <span class="network-logo">
-              <img width="35px" src="{{ networkLogoFacebook }}" />
+              <img class="fb_insta" src="{{ nerworkLogoFb_insta }}" />
```

Comment:
try to change name of class from fb_insta to fb-insta
for the variable notation try using lowerCaseUpperCase ( networkLogoFacebookInsta )